src/Pact/Types/Info.hs

```diff
@@ -58,6 +59,16 @@ data Parsed = Parsed {
   _pLength :: Int
   } deriving (Eq,Show,Ord,Generic)
 
+instance Arbitrary Parsed where
```

Comment:
So, I guess `Info` shows up in `PactValue` somewhere (guards?) -- that's too bad. What needs it?

Reply:
Yeah `Info` is used by`QualifiedName` and `BareName`. 
These are then used by `Name->UserGuard->Guard`. 
"Guard" is used by both PactValue and Namespace.

`Namespace` is not used in any other PactValue that I can see, but it has a SizeOf instance because it's used in the Gas Model's `ReadValue` and `WriteValue` data types because we're writing `Namespace` to the db.

NOTE: SizeOf `Info` == 0 because we didn't want to have a blow up in price.

I'm enforcing that all data types that implement the SizeOf class must also implement the Arbitrary class. This restriction was mainly a necessity for when I was implementing the Arbitrary class instance so I didn't miss any important ones.